libcxx/test/std/experimental/simd/simd.nonmembers/simd_alg.pass.cpp

```diff
@@ -0,0 +1,95 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03, c++11, c++14
+
+// <experimental/simd>
+//
+// [simd.nonmembers]
+// template<class T, class Abi> simd<T, Abi> min(const simd<T, Abi>& a, const simd<T, Abi>& b) noexcept;
+// template<class T, class Abi> simd<T, Abi> max(const simd<T, Abi>& a, const simd<T, Abi>& b) noexcept;
+// template<class T, class Abi> pair<simd<T, Abi>, simd<T, Abi>> minmax(const simd<T, Abi>& a, const simd<T, Abi>& b) noexcept;
+// template<class T, class Abi> simd<T, Abi> clamp(const simd<T, Abi>& v, const simd<T, Abi>& lo, const simd<T, Abi>& hi);
+
+#include "../test_utils.h"
+#include <cassert>
+#include <experimental/simd>
+
+namespace ex = std::experimental::parallelism_v2;
+
+struct CheckSimdMin {
+  template <class _Tp, class SimdAbi>
+  void operator()() {
+    const ex::simd<_Tp, SimdAbi> lhs([](_Tp i){return i;});
+    const ex::simd<_Tp, SimdAbi> rhs(static_cast<_Tp>(5));
+
+    ex::simd<_Tp, SimdAbi> result_min_simd = ex::min(lhs, rhs);
+    for (size_t i = 0; i < result_min_simd.size(); ++i) {
+        assert(result_min_simd[i] == std::min(lhs[i], rhs[i]));
```

Comment:
Use `assert_simd_value_correct` to check. Please check all tests for such problems. I will not mark them again